wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/WaitCommand.java

```diff
@@ -30,6 +34,16 @@ public WaitCommand(double seconds) {
     SendableRegistry.setName(this, getName() + ": " + seconds + " seconds");
   }
 
+  /**
+   * Creates a new WaitCommand. This command will do nothing, and end after the specified duration.
+   *
+   * @param time the time to wait
+   */
+  public WaitCommand(Measure<Time> time) {
+    m_duration = time.in(Second);
+    SendableRegistry.setName(this, getName() + ": " + m_duration + " seconds");
```

Comment:
I'm not sure why, considering we do the conversion anyways, the current way we just save it being done here

Reply:
I'm not 100% sure what you mean about "[saving] it being done here"- Either way, the conversion will be done once.
The point about delegating to the other constructor to avoid duplicating the logic, which means that if we make a change, we don't need to remember to change both constructors.

Reply:
Ok that makes sense, I'll change it

Reply:
In your latest commits you missed this suggestion. In fact, instead of copying the existing implementations you should _call_ the existing implementation with `time.in(Second)`.

Thanks!

Reply:
@spacey-sooty